.gitlab/dev_container_deploy/docker_linux.yml

```diff
@@ -264,6 +283,39 @@ qa_agent:
     IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-arm64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-win1809-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-winltsc2022-amd64
     IMG_DESTINATIONS: agent:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}
 
+qa_agent_ot:
```

Comment:
nit: We use `ot_agent` throughout this PR but `agent_ot` here. Do we want to consider naming this `qa_ot_agent`?

Reply:
I did this to be consistent with the naming convention in the file, where there's a separate section named `qa_agent`. If someone is grepping I'd like them to find our sections too, hence why I did this. I think I'd rather keep this as-is, but if you have a strong opinion, I'm also fine with changing this.

Reply:
Nope that makes sense to me, thanks! Will resolve these comments